sdk/python/feast/client.py

```diff
@@ -73,7 +73,20 @@
 
 _logger = logging.getLogger(__name__)
 
-CPU_COUNT = os.cpu_count()  # type: int
+
+def get_max_cpu_workers() -> int:
```

Comment:
Can we move this to another file in the package?

Reply:
Do you have any package suggestions? 

To be honest, I am new to the python code structure standard(if there is any), so I keep the function on the same place as it was before

Reply:
Please come up with proposals

Reply:
Because this is only one function According [Python Packaging Authority](https://www.pypa.io/en/latest/) and their sample project pip https://github.com/pypa/pip/tree/master/src/pip/_internal/utils

We can create have `utils` folder. Will create `utils` folder and inside it, there would be `os.py` which will  contain this function

Most of the other references allow it to be this way first because it is still 1 small function and only getting used in 1 place. When we have multiple functions then it would be a good time to think about the structure

Do let me know which one you're good, I am happy with any of the options

Reply:
How about we make its access private and move it to the bottom of the file?

Reply:
cool, I am happy with that as well

Reply:
This is done